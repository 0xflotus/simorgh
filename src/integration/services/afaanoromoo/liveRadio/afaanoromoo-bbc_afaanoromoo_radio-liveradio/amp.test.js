/**
 * @service afaanoromoo
 * @pathname /afaanoromoo/bbc_afaanoromoo_radio/liveradio
 */

import runAmpTests from '../../../../pages/liveRadio/ampTests';

describe('AMP afaanoromoo liveRadio', runAmpTests);
