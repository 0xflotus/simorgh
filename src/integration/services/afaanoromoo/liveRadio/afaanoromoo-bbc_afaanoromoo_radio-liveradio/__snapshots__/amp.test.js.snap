// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AMP afaanoromoo liveRadio A11y I can see the skip to content link 1`] = `"Qabiyyeetti darbi"`;

exports[`AMP afaanoromoo liveRadio AMP Consent config 1`] = `
Object {
  "consents": Object {
    "user-consent": Object {
      "promptIfUnknownForGeoGroup": "eea",
      "promptUI": "consent-prompt",
    },
  },
  "policy": Object {
    "default": Object {
      "timeout": Object {
        "fallbackAction": "reject",
        "seconds": 0,
      },
      "waitFor": Object {
        "user-consent": Array [],
      },
    },
  },
}
`;

exports[`AMP afaanoromoo liveRadio AMP Geo config 1`] = `
Object {
  "ISOCountryGroups": Object {
    "eea": Array [
      "at",
      "ax",
      "be",
      "bg",
      "bl",
      "cy",
      "cz",
      "de",
      "dk",
      "ea",
      "ee",
      "es",
      "fi",
      "fr",
      "gb",
      "gf",
      "gg",
      "gi",
      "gp",
      "gr",
      "hr",
      "hu",
      "ic",
      "ie",
      "im",
      "is",
      "it",
      "je",
      "li",
      "lt",
      "lu",
      "lv",
      "mf",
      "mq",
      "mt",
      "nc",
      "nl",
      "no",
      "pf",
      "pl",
      "pm",
      "pt",
      "re",
      "ro",
      "se",
      "si",
      "sj",
      "sk",
      "tf",
      "va",
      "wf",
      "yt",
    ],
  },
}
`;

exports[`AMP afaanoromoo liveRadio Analytics ATI 1`] = `
Object {
  "requests": Object {
    "base": "https://logws1363.ati-host.net?",
    "pageview": "\${base}s=598343&s2=2&p=afaanoromoo.bbc_afaanoromoo_radio.liveradio.page&r=\${screenWidth}x\${screenHeight}x\${screenColorDepth}&re=\${availableScreenWidth}x\${availableScreenHeight}&hl=\${timestamp}&lng=\${browserLanguage}&x1=[urn:bbc:ares:ws_media:live:bbc_afaanoromoo_radio]&x2=[amp]&x3=[news-afaanoromoo]&x4=[om]&x5=[\${sourceUrl}]&x6=[\${documentReferrer}]&x7=[player-live]&x8=[simorgh]&x9=[Dhaggeeffadhaa+-+BBC+News+Afaan+Oromoo]&ref=\${documentReferrer}",
  },
  "transport": Object {
    "beacon": false,
    "image": true,
    "xhrpost": false,
  },
  "triggers": Object {
    "trackPageview": Object {
      "on": "visible",
      "request": "pageview",
    },
  },
}
`;

exports[`AMP afaanoromoo liveRadio Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/afaanoromoo"`;

exports[`AMP afaanoromoo liveRadio Footer I can see the footer copyright and external linking text 1`] = `"Â© 2020 BBC. Qabiyyeewwan maddawwan alaa irraa ta'aniif BBCn itti gaafatamaa miti. Gara geessituu alaatti akkaataa itti hojjennu dubbisi."`;

exports[`AMP afaanoromoo liveRadio Header I can see the branding 1`] = `"BBC News, Afaan Oromoo"`;

exports[`AMP afaanoromoo liveRadio I can see the headline 1`] = `"Dhaggeeffadhaa"`;

exports[`AMP afaanoromoo liveRadio I can see the summary 1`] = `"Sagantaawwan keenya"`;

exports[`AMP afaanoromoo liveRadio Media player embed A11y I can read the media player title 1`] = `"Audio player"`;

exports[`AMP afaanoromoo liveRadio Media player embed I can see the media player embed 1`] = `"https://polling.test.bbc.co.uk/ws/av-embeds/media/bbc_afaanoromoo_radio/liveradio/om/amp?morph_env=live"`;

exports[`AMP afaanoromoo liveRadio SEO Canonical link 1`] = `"http://localhost:7080/afaanoromoo/bbc_afaanoromoo_radio/liveradio"`;

exports[`AMP afaanoromoo liveRadio SEO Dir attribute 1`] = `"ltr"`;

exports[`AMP afaanoromoo liveRadio SEO FB admins 1`] = `"100004154058350"`;

exports[`AMP afaanoromoo liveRadio SEO FB app ID 1`] = `"1609039196070050"`;

exports[`AMP afaanoromoo liveRadio SEO Heading level 1 1`] = `"Dhaggeeffadhaa"`;

exports[`AMP afaanoromoo liveRadio SEO Lang attribute 1`] = `"om"`;

exports[`AMP afaanoromoo liveRadio SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "RadioChannel",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/afaanoromoo.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/afaanoromoo/bbc_afaanoromoo_radio/liveradio",
    "@type": "WebPage",
    "name": "Dhaggeeffadhaa",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/afaanoromoo.png",
      "width": 1024,
    },
    "name": "BBC News Afaan Oromoo",
    "publishingPrinciples": "https://www.bbc.com/afaanoromoo/institutional-49281861",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/afaanoromoo.png",
  "url": "http://localhost:7080/afaanoromoo/bbc_afaanoromoo_radio/liveradio",
}
`;

exports[`AMP afaanoromoo liveRadio SEO OG description 1`] = `"Sagantaawwan keenya"`;

exports[`AMP afaanoromoo liveRadio SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/afaanoromoo.png"`;

exports[`AMP afaanoromoo liveRadio SEO OG image alt 1`] = `"BBC News Afaan Oromoo"`;

exports[`AMP afaanoromoo liveRadio SEO OG locale 1`] = `"om-ET"`;

exports[`AMP afaanoromoo liveRadio SEO OG site name 1`] = `"BBC News Afaan Oromoo"`;

exports[`AMP afaanoromoo liveRadio SEO OG title 1`] = `"Dhaggeeffadhaa - BBC News Afaan Oromoo"`;

exports[`AMP afaanoromoo liveRadio SEO OG type 1`] = `"website"`;

exports[`AMP afaanoromoo liveRadio SEO OG url 1`] = `"http://localhost:7080/afaanoromoo/bbc_afaanoromoo_radio/liveradio"`;

exports[`AMP afaanoromoo liveRadio SEO Page title 1`] = `"Dhaggeeffadhaa - BBC News Afaan Oromoo"`;

exports[`AMP afaanoromoo liveRadio SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`AMP afaanoromoo liveRadio SEO Twitter card 1`] = `"summary_large_image"`;

exports[`AMP afaanoromoo liveRadio SEO Twitter creator 1`] = `"@BBCNews"`;

exports[`AMP afaanoromoo liveRadio SEO Twitter description 1`] = `"Sagantaawwan keenya"`;

exports[`AMP afaanoromoo liveRadio SEO Twitter image alt 1`] = `"BBC News Afaan Oromoo"`;

exports[`AMP afaanoromoo liveRadio SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/afaanoromoo.png"`;

exports[`AMP afaanoromoo liveRadio SEO Twitter site 1`] = `"@BBCNews"`;

exports[`AMP afaanoromoo liveRadio SEO Twitter title 1`] = `"Dhaggeeffadhaa - BBC News Afaan Oromoo"`;
